### ISSUE TITLE
Hive Vancouver (HLC)

### ISSUE THREAD

**Issue by ldecoursy** [MozillaHive/HiveGlobal#2]
Driver: @simonaramkisson
Accountable: @ldecoursy
Responsible: @chrislarry33
Informed: Meghan, Kathryn
# Main Contacts: Helen Lee and Dethe Ezra

(Kat Braybrooke until she leaves for Europe)
[add contact info]
http://hivevan.org/

**Comment by ldecoursy** [comment]
# March 2, 2015

Helen, Dethe and Kat (she is in town for a little while at the moment) are still keeping the momentum alive for Hive Vancouver by co-chairing as Education Chair on the Vancouver Maker Foundation Board. 
They are currently planning to hold Maker Education meet-ups, the first one coming up next week at the Mozilla office. 

Currently looking for funding to hire a full time person to coordinate Hive Vancouver (the team connected with Lynn and Kathryn about provincial and national funding opportunities)

**Event Update**
Major event is next week’s Maker Education meet-up and I asked the team to speak about it on this month’s Hive Community Call

**Support Requested and Provided**
Main priority is funding and engaging potential new members

**Comment by simonaramkisson** [comment]
Update from Helen:

Just wanted to follow up on the Hive Global call last week.  Thanks again for the invitation and opportunity to share our experiences on the call.   Kat, Dethe & I have discussed amongst ourselves and decided that amongst the 3 of us, we simply don't have the capacity to maintain Hive Vancouver, its current web accounts (website, mailing lists & Twitter) or organize any Hive-related local events.  

We've managed to maintain the Hive partnerships made during the Pop-Up by partnering up with Vancouver Maker Foundation, and becoming part of their Maker Education component.  They are an enthusiastic, active local group that is able to provide us with support and connections, so we are able to engage all the Hive partners via their Maker Education events.  We will continue maintaining these partnerships via Maker Foundation.

In the meantime, we'd like to hand over all Hive Vancouver virtual accounts onto you and ask you to take ownership of these accounts, since we just don't have the capacity to manage them at this time.  These accounts were setup via Mozilla staff so I'm sure they can transition you over somehow on the back-end (sorry but I'm not fluent in the technical details).  

Thank you for all of your support and great work!  Please don't hesitate to contact us if you have any additional questions.

Cheers,

**Comment by chrislarry33** [comment]
Note: Concept paper that would potentially fund this work going into McConnell Friday

Copey here: https://docs.google.com/document/d/1OZ2fvJIKYpfVvDrINXJ5q7pvAz_YQwglaigWZM00-tU/edit

@simonaramkisson I'd like to connect on this one more. Do they have a suggestion on who might be a new local contact? Do you know someone we could recruit. Also would like to know more about this Maker Foundation.

I understand what happened here, but its a shame

**Comment by simonaramkisson** [comment]
@chrislarry33 Helen Lee is still in place and is the best driver but is only a rep and doesn't have massive capacity. She knows the landscape but because Dethe and Kat is no longer with Mozilla and Helen is a Rep but not working on this full-time. Helen and I have followed up with local partners but she is unable to keep up with demand. Hive Vancouver has a lot of potential!

**Comment by chrislarry33** [comment]
Yeah @simonaramkisson , love Helen (was an intern with Hive NYC!) but she needs help, would love to find another local person to back her up.

**Comment by simonaramkisson** [comment]
Emma Irwin was in Scotland with me this week and we discussed opportunities to jump start Hive Vancouver with Helen. The discussion was focused on maintaining relationship with Maker Foundation and how we involve reps more in work.
